dominion/dominion_ui/tests.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
Empty files maybe just don't them it in here until we add logic.

- https://github.com/CodeForAfrica/HURUmap-apps/pull/135/files#diff-0edaf497805b6fd7f0b48727a7bd32bcR6
- https://github.com/CodeForAfrica/HURUmap-apps/pull/135/files#diff-a713d31f2e54514f01aa62cf4f910550R6
- https://github.com/CodeForAfrica/HURUmap-apps/pull/135/files#diff-f88e05aba4295cb63ba614c48f2a35bcR6